retrofit2/src/main/java/com/linecorp/armeria/client/http/retrofit2/ArmeriaRetrofitBuilder.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.http.retrofit2;
+
+import static com.linecorp.armeria.client.http.retrofit2.ArmeriaCallFactory.GROUP_PREFIX;
+import static java.util.Objects.requireNonNull;
+
+import java.net.URI;
+import java.util.concurrent.Executor;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+
+import com.linecorp.armeria.client.ClientFactory;
+import com.linecorp.armeria.client.ClientOptions;
+import com.linecorp.armeria.client.Clients;
+import com.linecorp.armeria.client.http.HttpClient;
+import com.linecorp.armeria.common.Scheme;
+import com.linecorp.armeria.common.SerializationFormat;
+import com.linecorp.armeria.common.SessionProtocol;
+
+import okhttp3.HttpUrl;
+import retrofit2.Call;
+import retrofit2.CallAdapter;
+import retrofit2.Callback;
+import retrofit2.Converter;
+import retrofit2.Retrofit;
+import retrofit2.Retrofit.Builder;
+
+/**
+ * A helper class for creating a new {@link Retrofit} instance with {@link ArmeriaCallFactory}.
+ * For example,
+ * <pre>{@code
+ *
+ * Retrofit retrofit = new ArmeriaRetrofitBuilder()
+ *     .baseUrl("http://localhost:8080/")
+ *     .build();
+ *
+ * MyApi api = retrofit.create(MyApi.class);
+ * Response&lt;User&gt; user = api.getUser().execute();
+ * }
+ * </pre>
+ *
+ * <p>ArmeriaRetrofitBuilder even supports EndpointGroup, so you can create Retrofit like below,
```

Comment:
Please use `{@link ...}` tags for the class names - ArmeriaRetrofitBuilder, EndpointGroup and Retrofit